website/content/en/docs/contribution-guidelines/plugins.md

```diff
@@ -0,0 +1,80 @@
+
+Then, the specific files for each language are scaffolded for the language's plugins. In this way, the SDK CLI tool can provide custom plugins and helpers which are common and useful for all language types. To check the common base, you can also run:
+
+```sh
+operator-sdk init --plugins=kustomize
+```
+
+Also, see the topic [Language-based Plugins][kb-language-plugins] to understand how it works.
+
+### Common scaffolds 
+
+Following the common default scaffolds for the projects which are built with SDK.
+
+| File/Directory | Description | 
+| ------ | ----- |
+|  Dockerfile |  Defines the operator(manager) image |  
+|  Makefile |  Provides the helpers and options for the users. (e.g. `make bundle` which generate/update the OLM [bundle][bundle] manifests ) |  
+|  PROJECT |  Project configuration. Stores the data used to do the scaffolds. For further information see [Project Config][kb-project] | 
+|  bundle.Dockerfile | Docker image which is used to provide the helpers to integrate the project with OLM. (e.g. [operator-sdk run ./bundle][sdk-cli-run-bundle]) | 
+|  config/ |  Directory which has all [kustomize's][kustomize] manifest to configure and test the project | 
+
+You can check the [Project Layout][project-layout] to better understand the files and directories scaffold by SDK CLI, which are common or not for each Language-type. 
+
+## Custom Plugins
+
+By default, some plugins are used by SDK to add the required changes in the scaffolds to provide its features which currently are: 
+
+- [manifests.sdk.operatorframework.io][plugin-manifest]: responsible for performing all required changes in the scaffolds and providing the helpers to allow the projects to be integrated with OLM. 
+- [scorecard.sdk.operatorframework.io][plugin-scorecard]: responsible to scaffold all files and helpers required to provide the [Scorecard][scorecard] feature
+
+### Optional/custom plugins
+
+Users also can use custom plugins when they execute the SDK CLI sub-commands as a helper for its options which would like:
+
+```sh
+operator-sdk create api --plugins="go/v3,declarative"
+```
+
+The above example will scaffold a custom code in the controllers after when an api is created to allow the users develop solutions using the [kubebuilder declarative pattern][kubebuilder-declarative-pattern]. (e.g [default scaffold][default-scaffold] versus [example][kubebuilder-declarative-pattern-example]).
+
+Note that custom plugins which are called via the init sub-command, will be added in the `layout` field of the PROJECT file and will work such as global plugins. Then, it means that for any sub-command executed; they will also be called. 
```

Comment:
Note that custom plugins are called via the init sub-command to work as global plugins, and will be added in the `layout` field of the PROJECT file. Any sub-command executed will then also be called.

Reply:
done.